include/mpm_explicit_twophase.h

```diff
@@ -0,0 +1,65 @@
+#ifndef MPM_MPM_EXPLICIT_TWOPHASE_H_
+#define MPM_MPM_EXPLICIT_TWOPHASE_H_
+
+#include "mpm_base.h"
+
+namespace mpm {
+
+//! MPMExplicit class
+//! \brief A class that implements the fully explicit one phase mpm
+//! \details A single-phase explicit MPM
+//! \tparam Tdim Dimension
+template <unsigned Tdim>
+class MPMExplicitTwoPhase : public MPMBase<Tdim> {
```

Comment:
Hmmm, I guess 2 phase explicit should be derived from mpm_explicit, not from mpm_base to reduce code duplication. What do you think @srhgk2 @kks32?